Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM registry.access.redhat.com/ubi9/ubi@sha256:9e6a89ab2a9224712391c77fab2ab010
 FROM docker.io/library/golang:1.20.0-bullseye as golang_120
 FROM docker.io/library/golang:1.21.0-bullseye as golang_121
 FROM docker.io/library/node:22.7.0-bullseye as node
+FROM docker.io/library/ruby:3.1.6-bullseye as ruby
```

Comment:
Dockerfile changes should be separate from core logic, so should go to a different commit (in general, see my other comment below).